### PR TITLE
adding helloworld example to finish out quickstart

### LINKED ISSUE #42: How do you configure the StorageClass for the vagrant deployment?
While using the vagrant setup, both @screeley and I are running into the problems encountered with https://github.com/gluster/gluster-kubernetes/issues/24. The GlusterFS provisioner has undergone a lot of changes in Kubernetes lately so the first thing I did was ascertain the version the kubernetes version, which is 1.4.5. 

According to history of the provisioning README at the time 1.4.5 shipped (https://github.com/kubernetes/kubernetes/blob/1d527194656bad6a0f191f9fc6160bf7e931cf09/examples/experimental/persistent-volume-provisioning/README.md) the following storage class and claim should work, but they don't. Any idea @jarrpa @humblec ?

[root@master deploy]# cat gluster-sc.yaml 
apiVersion: storage.k8s.io/v1beta1
kind: StorageClass
metadata:
  name: glusterfs
provisioner: kubernetes.io/glusterfs
parameters:
  resturl: "http://10.36.0.0:8080"
  restuser: "admin"
  secretNamespace: "default"
  secretName: "heketi-service-account-token-7ne36"

[root@master deploy]# cat gluster-claim.yaml 
{
  "kind": "PersistentVolumeClaim",
  "apiVersion": "v1",
  "metadata": {
    "name": "test-claim",
    "annotations": {
        "volume.beta.kubernetes.io/storage-class": "glusterfs"
    }
  },
  "spec": {
    "accessModes": [
      "ReadWriteMany"
    ],
    "resources": {
      "requests": {
        "storage": "5Gi"
      }
    }
  }
}

### PR DESCRIPTION
**Pull request by screeley44**
Fixes #42 

Adding correct yaml for this version of vagrant quickstart and an examples section including a HelloWorld to finish the quickstart thought

### CODE REVIEW

**Comment by screeley44** [comment]
@obnoxxx @jarrpa - PTAL

File: README.md [hunk 1]
```diff
@@ -59,6 +59,20 @@ For ease of use in the the vagrant setup, we recommend you run the following:
...
+```bash
+$ kubectl get nodes
+NAME      STATUS    AGE
+master    Ready     22h
+node0     Ready     22h
+node1     Ready     22h
+node2     Ready     22h
+```
+
+#### NOTE: To see the version of Kubernetes (which will change based on latest official releases) simply do _kubectl version_
```
Comment (jarrpa) [inline README.md]: Stylistic preferences: `_kubectl version_` should be ``kubectl version``, and `#### NOTE:` should be replaced with  `***NOTE:***` or some non-heading equivalent.

File: README.md [hunk 2]
```diff
@@ -68,14 +82,57 @@ $ ./gk-deploy -g
 If you already have GlusterFS deployed in your cluster, you do not need the
 `-g` option.
 
-heketi should now be installed and ready to go. For ease of testing, we
-recommend the following:
+
+After this completes, to aid in testing and following the rest of the examples, the following export 
+should be run and will set the _HEKETI_CLI_SERVER_ env variable and will also give an
```
Comment (jarrpa) [inline README.md]: Similarly, `_HEKETI_CLI_SERVER_` should be ``HEKETI_CLI_SERVER``.

File: README.md [hunk 3]
```diff
@@ -68,14 +82,57 @@ $ ./gk-deploy -g
...
 
 ```bash
 $ export HEKETI_CLI_SERVER=$(kubectl describe svc/heketi | grep "Endpoints:" | awk '{print "http://"$2}')
+
+$ echo $HEKETI_CLI_SERVER
+http://10.42.0.0:8080
+
+```
+
+
```
Comment (jarrpa) [inline README.md]: Remove excessive newlines above.

File: README.md [hunk 4]
```diff
@@ -68,14 +82,57 @@ $ ./gk-deploy -g
...
+glusterfs-node0-2509304327-vpce1   1/1       Running             0          1d
+glusterfs-node1-3290690057-hhq92   1/1       Running             0          1d
+glusterfs-node2-4072075787-okzjv   1/1       Running             0          1d
+heketi-3017632314-yyngh            1/1       Running             0          1d
+
+```
+
+Notice that we have 3 gluster pods running and 1 heketi pod running
+
+
```
Comment (jarrpa) [inline README.md]: Similarly here, remove excessive newlines, and the two bash blocks can be merged together.

File: docs/examples/hello_world/README.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+```
+kubectl get endpoints
+NAME                       ENDPOINTS                                            AGE
+heketi                     10.42.0.0:8080                                       22h
+heketi-storage-endpoints   192.168.10.100:1,192.168.10.101:1,192.168.10.102:1   22h
+kubernetes                 192.168.10.90:6443                                   23h
+```
+
+##### NOTE: EndPoints define the GlusterFS cluster, for version 1.4.X this is a required parameter for the StorageClass (Parameters are likely to change in later versions)
```
Comment (jarrpa) [inline docs/examples/hello_world/README.md]: Don't use sub-heading notation for notes, bold or italicize or something similar.

**Comment by jarrpa** [comment]
Some of my review comments lost the fact that I was requesting the use of backticks instead of underscores for inline commands/variables.

**Comment by screeley44** [comment]
@jarrpa - fixed requested changes

File: README.md (hunk 4 above)
Comment (screeley44) [inline README.md]: done

File: docs/examples/hello_world/README.md (hunk 5 above)
Comment (screeley44) [inline docs/examples/hello_world/README.md]: done

**Comment by jarrpa** [comment]
There's still a few small stylistic changes, but nothing worth a holdup. I can submit a quick follow up PR to address them. LGTM. :)

**Pull request merged**